### PR TITLE
[LTC] Autogen sigmoid and rsqrt for TS backend

### PR DESCRIPTION
**Pull request by ramiro050**
Adds `full_codegen` support for `sigmoid` and `rsqrt` ops using the same approach used in #64104 to add codegen support for `cos`.

Verified with the following script:

```python
import torch
import lazy_tensor_core as ltc
from lazy_tensor_core.debug import metrics

ltc._LAZYC._ltc_init_ts_backend()
device = 'lazy'
dtype = torch.float32

x = torch.randn((2,3,4), device=device, dtype=dtype)

def computation(x):
    return x.sigmoid() * x.rsqrt()

computation(x)
print(metrics.metrics_report())
```

Output:

```
Metric: IrValueTensorToDataHandle
  TotalSamples: 1
  Accumulator: 009.717us
  Percentiles: 1%=009.717us; 5%=009.717us; 10%=009.717us; 20%=009.717us; 50%=009.717us; 80%=009.717us; 90%=009.717us; 95%=009.717us; 99%=009.717us
Counter: CreateLtcTensor
  Value: 7
Counter: DestroyLtcTensor
  Value: 6
Counter: aten::normal_
  Value: 1
Counter: lazy::_copy_from
  Value: 1
Counter: lazy::_copy_from_and_resize
  Value: 1
Counter: lazy::mul
  Value: 2
Counter: lazy::rsqrt
  Value: 2
Counter: lazy::sigmoid
  Value: 2
```

### CODE REVIEW

**Review by alanwaketan — COMMENTED** [review]
Overall LGTM.

Few other comments:
1. Normally, we would love to have test plan in the commit message, like:
`test/cpp/build/test_ptltc --gtest_filter=AtenLtcTsTensorTest.TestClamp*`
AtenLtcTsTensorTest is the ops test that we have for LTC.

2. Last but not least, can you rebase to our latest lazy_tensor_staging branch which just enable the new CI?

**Review by alanwaketan — APPROVED** [review]
LGTM. Thanks for improving code-gen.
